master/internal/command/command_service.go

```diff
@@ -67,7 +67,7 @@ func (cs *CommandService) RestoreAllCommands(
 		Where("task.task_id = command_snapshot.task_id").
 		Scan(ctx)
 	if err != nil {
-		return fmt.Errorf("failed to remake commands: %w", err)
+		cs.syslog.Errorf("failed to remake commands: %s", err)
```

Comment:
need to return. also maybe apply the same idea on line 76.

Reply:
Why do we need to return? This fix is something Nick suggested in response to https://hpe-aiatscale.slack.com/archives/C04C9JXB1C2/p1700232278662739

Reply:
Because if err != nil, snapshots will be nil and the rest of the flow is nonsense/just luck that there is not a nil deref.